WebRoot/js/zimbraMail/abook/view/ZmContactSearch.js

```diff
@@ -428,23 +418,46 @@ ZmContactSearchListView.prototype._getHeaderList =
 function() {
 	var headerList = [];
 	headerList.push(new DwtListHeaderItem({field:ZmItem.F_TYPE, width:ZmMsg.COLUMN_WIDTH_FOLDER_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name, width:ZmMsg.COLUMN_WIDTH_NAME_CN}));
-	headerList.push(new DwtListHeaderItem({field:ZmItem.F_EMAIL, text:ZmMsg.email}));
-
+	headerList.push(new DwtListHeaderItem({field:ZmItem.F_NAME, text:ZmMsg._name}));
 	return headerList;
 };
 
+// Overriding the DwtListView.prototype.createHeaderHtml as we donot need a header, here.
+
+ZmContactSearchListView.prototype.createHeaderHtml =
+function() {
```

Comment:
It will be better if you maintain parameters of method you are overriding. Though we are not using it here, it makes sense to have to IMO.

For instance: `ZmContactSearchListView.prototype.createHeaderHtml(defaultColumnSort, isColumnHeaderTableFixed)`

Also not sure, if it's okay to return `null` from this method. Empty method looks weird to me. 

Take your call.

Thanks